sklearn/tests/test_common.py

```diff
@@ -416,45 +399,6 @@ def test_check_param_validation(estimator):
     check_param_validation(name, estimator)
 
 
-@pytest.mark.parametrize(
-    "Estimator",
-    [
-        AffinityPropagation,
-        Birch,
-        MeanShift,
-        KNeighborsClassifier,
-        KNeighborsRegressor,
-        RadiusNeighborsClassifier,
-        RadiusNeighborsRegressor,
-        LabelPropagation,
-        LabelSpreading,
-        OPTICS,
-        SpectralClustering,
-        LocalOutlierFactor,
-        LocallyLinearEmbedding,
-        Isomap,
-        TSNE,
-    ],
```

Comment:
Is the idea now that all the estimators are tested, and not just these?

Reply:
Yes. I checked and they all pass anyway, so I don't see why we wouldn't test all of them.